httpapi/health/health.go

```diff
@@ -0,0 +1,47 @@
+package health
+
+import (
+	"net/http"
+
+	"github.com/centrifuge/go-centrifuge/version"
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+)
+
+// config defines required methods for Health handler
+type config interface {
+	GetNetworkString() string
+}
+
+// handler handles the Health APIs
+type handler struct {
+	c config
+}
+
+// Pong is the response for Ping
+type Pong struct {
+	Version string `json:"version"`
+	Network string `json:"network"`
+}
+
+// Ping responds with node version and network
+
+// @summary Health check for Node
+// @description returns node version and network
+// @id ping
+// @produce json
+// @success 200 {object} health.Pong
+// @router /ping [get]
+func (h handler) Ping(w http.ResponseWriter, r *http.Request) {
+	render.Status(r, http.StatusOK)
+	render.JSON(w, r, Pong{
```

Comment:
How much control we have over the JSON marshaller?

Reply:
render.JSON uses std lib's json.Marshaller. If we need to provide a custom marshaller, we may choose to write our own func like render.JSON or give them a PR to allow custom marshaller